### PR TITLE
Use a unified naming scheme for jobs

### PR DESCRIPTION
**Pull request by dgn**

### CODE REVIEW

File: prow/config/postsubmits.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ postsubmits:
         - -m bump-rpm
 
   Maistra/istio-operator:
-  - name: update-rpm
+  - name: istio-operator_update-rpm
```
Comment (jwendell) [inline prow/config/postsubmits.yaml:72]: just the prefix `operator` would be easier to read and do the same job, agree?

File: prow/config/presubmits.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 presubmits:
   Maistra/maistra.github.io:
-  - name: lint
+  - name: maistra.github.io_lint
```
Comment (jwendell) [inline prow/config/presubmits.yaml:3]: what about just `maistra.io`?

File: prow/config/presubmits.yaml [hunk 3]
```diff
@@ -150,7 +150,7 @@ presubmits:
         securityContext:
           privileged: true
   Maistra/istio:
-  - name: maistra-istio-1.2-unit
+  - name: istio_1.2-unit
```
Comment (jwendell) [inline prow/config/presubmits.yaml:153]: `s/istio/maistra/`?

**Comment by dgn** [comment]
@jwendell to your comments: the idea was that all jobs are prefixed with the repository name so they're more easily searched in the prow UI. I'm not sure having names that are slightly different to the repositories is good, could be confusing

**Pull request merged**